tests/src/test/scala/common/rest/WskRest.scala

```diff
@@ -1114,32 +1104,31 @@ class WskRestApi extends RunWskRestCmd with BaseApi {
                     expectedExitCode: Int = SUCCESS_EXIT,
                     cliCfgFile: Option[String] = None)(implicit wp: WskProps): RestResult = {
 
-    val parms = Map[String, JsValue]() ++ {
+    val parms = {
       basepathOrApiName map { b =>
         Map("basepath" -> b.toJson)
-      } getOrElse Map[String, JsValue]()
+      } getOrElse Map.empty
```

Comment:
is there a scala best practice preference (assuming a code block and method invocation implementations are equivalent)?

`basepathOrApiName.map(b => Map("basepath" -> b.toJson)).getOrElse(Map.empty)`

Reply:
There's none I believe, I tend to use your way of of writing @mdeuser because it feels more explicit, but that's just me. The compiler also tends to get funny with infix notation in some edge-cases, so I usually avoid that alltogether.